components.yaml

```diff
@@ -35,6 +35,6 @@ MAPL:
 GEOSgcm_GridComp:
   local: ./src/Components/GEOSldas_GridComp/@GEOSgcm_GridComp
   remote: ../GEOSgcm_GridComp.git
-  branch: develop  
+  branch: feature/borescan_merge_sm_peat  
```

Comment:
@biljanaorescanin I think this can go back to `develop` now as your branch went away with v1.14.1.